tcms/issuetracker/types.py

```diff
@@ -180,7 +180,7 @@ def report_issue_from_testexecution(self, execution, user):
             if not url.endswith('/'):
                 url += '/'
 
-            return url + '/issues/new?' + urlencode(args, True)
+            return url + 'issues/new?' + urlencode(args, True)
```

Comment:
Noticed a bug here during development. Github url gets an extra "/" in case of exception and url does not work.

Reply:
This is unrelated to the current changes so should be removed from this PR. You can submit this change independently.